docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -41,88 +41,147 @@ Example projects do not create an accompanying UI.
+
+    To build the example:
+
+    ```sh
+    npm run build
+    ```
     
-      This example uses `berkeley`, but the deploy alias name can be anything and does not have to match the network name.
-  
-    - Mina GraphQL API URL: `https://proxy.berkeley.minaexplorer.com/graphql`
+    - The `npm run build` command builds the TypeScript files in `sudoku/src` that contain the code for the smart contract.
+    - This build command compiles the TypeScript code into JavaScript in the `sudoku/build` directory.  
+
+1. Configure your zkApp:
+
+    ```sh
+    zk config
+    ```
+
+    The command prompts guide you to add a deploy alias to your project `config.json` file. 
+    
+1. Define a name for the deploy alias. 
+
+    For this example, use:
   
-    - Transaction fee to use when deploying: `0.1`
+    ```text
+    berkeley
+    ``` 
+      
+    The deploy alias name does not have to match the network name.
+
+1. Choose the target network:
+
+    ```text
+    Testnet
+    ``` 
+ 
+ 1. Set the Mina GraphQL API URL to deploy to: 
+
+    ```text
+    https://proxy.berkeley.minaexplorer.com/graphql
+    ``` 
   
-    - Account to pay transaction fees: Create a new fee payer pair
+1. Set the transaction fee to use when deploying: 
 
-1. When prompted to choose an account to pay transaction fees, select to use a different account:
+    ```text
+    0.1
+    ``` 
+ 
+1. When prompted to choose an account to pay transaction fees, select:
 
-  ```sh
-  Use a different account (select to see options)
-  ```
+    ```text
+    Use a different account (select to see options)
+    ```
 
-  If this is the first time you run the `zk config` command, you see these options:
+    If this is the first time you are running the `zk config` command, you see these options:
 
-  ```text
-  ❯ Recover fee payer account from an existing base58 private key
-  Create a new fee payer key pair
-  ```
+    ```text
+    ❯ Recover fee payer account from an existing base58 private key
+    Create a new fee payer key pair
+    ```
 
-The option to choose another account is shown only if the user has a cached fee payer account.
+    The option to choose another account is shown only if you have a cached fee payer account.
 
-1. Next, select to create a new fee payer key pair:  
+1. Select to create a new fee payer key pair:  
 
-  ```sh
-  Create a new fee payer key pair
-  NOTE: the private key will be stored in plain text on this computer.
-  ```
+    ```sh
+    Create a new fee payer key pair
+    NOTE: the private key will be stored in plain text on this computer.
+    ```
 
-1. When prompted, give an alias to your new fee payer key pair. For this example, use `sudoku`:
+    A fee payer account is a funded developer account that can always pay fees immediately.
 
-  ```sh
-  Create an alias for this account: sudoku
-  ```
+1. When prompted to create an alias for this account, give an alias to your new fee payer key pair:
```

Comment:
Do we want to give advice about appropriate for local testing only since the private key is in plain text? or more guidance on fee payer account?

Reply:
That's a good idea!